bee-message/src/serde.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+/// Helper macro to serialize types to string via serde.
+#[macro_export]
+macro_rules! string_serde_impl {
```

Comment:
This is me writing before thinking, but could it be possible to achieve the same using the `serde_with` attribute?

Reply:
Removing this 🙂